src/app/dashboard/_dashboard.scss

```diff
@@ -7,35 +8,114 @@ $paperCardBackgroundColor: #212121 !default;
 
 #dashboardContainer {
 	background-color: $dashboardContainerBackgroundColor;
+	height: 100%;
 
-	.userbar {
+	// typographic defaults
+	font-family: Roboto, sans-serif;
+	font-style: normal;
+	font-size: 1em;
+	line-height: 1.2em;
```

Comment:
i see you are using `rem` everywhere else (did you want to convert these)

also if this block of font code is going to be reused multiple times lets look at making it a mixin
that way we can just use `@include defaultFont();`  

or we can make it a class and include it in the scss by using `@extend .defaultFont;`